test/case/01_bootstrap_update

```diff
@@ -18,3 +18,18 @@ cd $repo
 export REPO_NAME=${0##*/}
 
 make update_boilerplate
+
+check_update 
+
+if [ $? -ne 0 ] ; then
+	return $?
+fi
+
+add_convention openshift/test_base_convention
+add_convention openshift/test_base
```

Comment:
A nonexistent convention should cause `update` to fail. The output looks promising:

```
Invalid convention directory: 'openshift/test_base'
make[1]: *** [Makefile:3: update_boilerplate] Error 1
make[1]: Leaving directory '/tmp/tmp.fOXGsNq9ug'
```

...but the overall test case isn't failing. We need to fix that.

[Later] There are a couple of pieces causing this. One is that the main driver needs to *not* use `source` [here](https://github.com/openshift/boilerplate/pull/11/files#diff-ad6c39a41f5fcfb2b3d89656bfb00755R28). I saw that yesterday but forgot to take it out. Another is that we need to capture errors [here](https://github.com/openshift/boilerplate/pull/11/files#diff-e1b2b21859e872e9af54745ed0a1b5c4R103) -- commented accordingly.

Reply:
Fixed via #14.